From 812334604afd884279d62c68d5f0b66274431efa Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 22 Feb 2020 22:37:46 +0200
Subject: [PATCH 084/105] Revert "try to fix align of network indicator"

This reverts commit f59ba6725a4a20f9a4f56ec4b432dbe75ebd9086.

	modified:   packages/SystemUI/res/layout/system_icons.xml
---
 packages/SystemUI/res/layout/system_icons.xml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/res/layout/system_icons.xml b/packages/SystemUI/res/layout/system_icons.xml
index 8e7b8b6bba4..ed0a6f66597 100644
--- a/packages/SystemUI/res/layout/system_icons.xml
+++ b/packages/SystemUI/res/layout/system_icons.xml
@@ -26,7 +26,6 @@
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
         android:gravity="start|center_vertical"
-				android:clipChildren="true"
         >
 
         <org.lineageos.internal.statusbar.NetworkTraffic
@@ -34,7 +33,7 @@
             android:layout_width="wrap_content"
             android:layout_height="match_parent"
             android:maxLines="2"
-            android:gravity="start|center_vertical" />
+            android:gravity="center_vertical|start" />
 
     </com.android.systemui.statusbar.LineageStatusBarItemHolder>
 
-- 
2.17.1

